src/components/Layout/Partials/SideBar.tsx

```diff
@@ -17,35 +17,8 @@ export class SideBar extends React.Component<undefined, undefined> {
                         </li>
                     </ul>
                 </nav>
-                <div className="social-list">
-                    <a href="#" className="social-list__item">
```

Comment:
Moving links and divs to another file is unduly.
You shall create one file for each svg and mount it like `<a><GitSvg></a>`